src/main/java/org/cascadebot/cascadebot/commands/music/SeekCommand.java

```diff
@@ -0,0 +1,67 @@
+package org.cascadebot.cascadebot.commands.music;
+
+import com.ibm.icu.text.MeasureFormat;
+import com.ibm.icu.util.Measure;
+import com.ibm.icu.util.ULocale;
+import net.dv8tion.jda.api.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.ParserUtils;
+import org.cascadebot.shared.Regex;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+public class SeekCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
```

Comment:
The command really needs to check for if a user is actually listening to the song

Reply:
It does 👀

Reply:
no it doesn't it checks if a song is playing but not if the user who ran it is in the same voice channel

Reply:
Ah gotcha, misunderstood mb